src/sidebar/components/new-note.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var events = require('../events');
+
+module.exports = {
+  controllerAs: 'vm',
+  //@ngInject
+  controller: function ($rootScope, annotationUI) {
```

Comment:
For consistency with other component modules, the controller should be a named function with the `module.exports = {...}` below it rather than an anonymous function.